### PR TITLE
nextcloud-client: Add recipe for 3.11.0

### PR DESCRIPTION
**Pull request by m199**
This is the initial release of the recipe.

### CODE REVIEW

**Comment by Begasus** [comment]
Quite some work to be done for the recipe, I'll do some further testing here, builds ok so far on 64bit, thanks!

File: net-misc/nextcloud-client/additional-files/nextcloud_client.rdef [hunk 1]
```diff
@@ -0,0 +1,29 @@
+resource app_version {
+	major  = 3,
+	middle = 11,
+	minor  = 0,
+	variety = B_APPV_FINAL,
+	internal = 0,
+	short_info = "Nextcloud Desktop Client",
```
Comment (humdingerb) [inline net-misc/nextcloud-client/additional-files/nextcloud_client.rdef:7]:
```suggestion
	short_info = "Nextcloud desktop client",
```

File: net-misc/nextcloud-client/additional-files/nextcloud_client.rdef [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	long_info = "Nextcloud Desktop Client"
```
Comment (humdingerb) [inline net-misc/nextcloud-client/additional-files/nextcloud_client.rdef:8]:
```suggestion
	long_info = "Client for syncing files with Nextcloud"
```

File: net-misc/nextcloud-client/nextcloud_client-3.11.0.recipe [hunk 3]
```diff
@@ -0,0 +1,143 @@
+SUMMARY="Desktop Syncing Client for Nextcloud"
```
Comment (humdingerb) [inline net-misc/nextcloud-client/nextcloud_client-3.11.0.recipe:1]:
```suggestion
SUMMARY="A syncing client for Nextcloud"
```

File: net-misc/nextcloud-client/nextcloud_client-3.11.0.recipe [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+DESCRIPTION="The Nextcloud Desktop Client is a tool to synchronize
```
Comment (humdingerb) [inline net-misc/nextcloud-client/nextcloud_client-3.11.0.recipe:2]:
```suggestion
DESCRIPTION="This Nextcloud client is a tool to synchronize files with a Nextcloud server.
```

Could do with listing some features etc...

File: net-misc/nextcloud-client/nextcloud_client-3.11.0.recipe [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+files from Nextcloud Server with your computer."
+HOMEPAGE="http://nextcloud.com"
```
Comment (humdingerb) [inline net-misc/nextcloud-client/nextcloud_client-3.11.0.recipe:4]:
```suggestion
HOMEPAGE="https://nextcloud.com"
```

**Comment by m199** [comment]
Begasus reworked the recipe and there was a new release in the meantime. I wasnt able to update this in a clean way. I close this PR and start a new one for 3.11.1. Thanks for working on this!

**Pull request closed without merging**